store/forms/update_bridge_type.go

```diff
@@ -0,0 +1,51 @@
+package forms
+
+import (
+	"github.com/manyminds/api2go/jsonapi"
+	"github.com/smartcontractkit/chainlink/store"
+	"github.com/smartcontractkit/chainlink/store/assets"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/smartcontractkit/chainlink/store/presenters"
+)
+
+// NewUpdateBridgeType initializes the form attributes with the existing
+// attributes from the bridge
+func NewUpdateBridgeType(store *store.Store, bridgeName string) (UpdateBridgeType, error) {
+	bt, err := store.FindBridge(bridgeName)
+	if err != nil {
+		return UpdateBridgeType{}, err
+	}
+
+	form := UpdateBridgeType{
+		store:                  store,
+		bridgeType:             bt,
+		URL:                    bt.URL,
+		Confirmations:          bt.Confirmations,
+		MinimumContractPayment: bt.MinimumContractPayment,
+	}
+	return form, nil
+}
+
+// UpdateBridgeType whitelists attributes that can be updated on the bridge
+type UpdateBridgeType struct {
+	store                  *store.Store
+	bridgeType             models.BridgeType
```

Comment:
```suggestion
```

Can we remove the `bridgeType` altogether?

1. Its values are redundant to the form object (`URL, Confirmations, MinimumContractPayment`)
2. You can just create a new instance in `Save` and `Marshal`, reducing shared state. You can still be DRY by placing it in a private method.

Reply:
@dimroc I wanted to error as soon as possible. If we want to keep the behavior of the early `not found error` in `NewUpdateBridgeType` we would have to do another find of `BridgeType` in `Save`.

What's your preference on where the `not found error` should happen?

Reply:
You could do `_, err := store.FindBridge(bridgeName)` in `NewUpdateBridgeType` and save a new `bt` in `Save`. Let's chat about it at standup.